src/LibraryUtilities.ts

```diff
@@ -788,7 +788,35 @@ export function findAndExpandItemByPath(pathToItem: ItemData[], allItems: ItemDa
 export function sortItemsByText(items: ItemData[]): ItemData[] {
     let sortedItems = items.sort(function (item1: ItemData, item2: ItemData) {
         return item1.text.localeCompare(item2.text);
-    })
+    });
 
     return sortedItems;
+}
+
+/**
+ * Get the path to expanded items.
+ *
+ * @param {ItemData} rootItem 
+ * The root item that is the parent of all the expanded/collapsed items.
+ * 
+ * For example, in the following tree, Item A, B are expanded, and A is the root item.
+ * This function will return [A, B, C] as the result, which represents a path to all expanded items.
+ * 
+ * - A
+ *   |- B
+ *      |- C
+ *      |- D
+ *   |- E
+ *   |- F
+ * 
+ */
+export function getPathToExpandedItemFromRootItem(rootItem: ItemData, itemPath: ItemData[] = []): ItemData[] {
```

Comment:
Do we really need to return `ItemData[]` or will `string[]` work as well? I think we need `ItemData[]` for its completeness, but just want to confirm.